config/initializers/valkyrie.rb

```diff
@@ -1,20 +1,30 @@
 # frozen_string_literal: true
 require_relative "figgy"
+
 Rails.application.config.to_prepare do
+  # Registers a storage adapter for a *NIX file system
+  # Binaries are persisted by invoking "mv" with access limited to read/write for owning users, and read-only for all others
+  # NOTE: "mv" may preserve the inode for the file system
```

Comment:
Can we add the use case for why `mv`? (Which is that this is typically used for pulling uploads from a temporary location such as a file upload from a form)

Reply:
Also why the chmod is important (if I remember, it was moving files to the derivative location and preserving the permissions and the image server couldn't read them.)